components/Tabs/src/Theming/MDCTabBar+MaterialTheming.m

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCTabBar+MaterialTheming.h"
+
+#import <CoreGraphics/CoreGraphics.h>
+#import <UIKit/UIKit.h>
+
+#import <MaterialComponents/MaterialColorScheme.h>
+#import <MaterialComponents/MaterialTypographyScheme.h>
+
+static const CGFloat kPrimaryThemeUnselectedOpacity = (CGFloat)0.74;
+static const CGFloat kSurfaceThemeUnselectedOpacity = (CGFloat)0.74;
```

Comment:
This should be 0.60 for Surface Theme.

Reply:
oops. Thanks for catching it. Addressed in 0add201006e24ba62ae4a717b90a6a7cc39e91af